src/routes/_components/dialog/components/MediaDialog.html

```diff
@@ -25,8 +25,24 @@
         </li>
       {/each}
     </ul>
+    {#if hasDescription }
+    <div class="media-description-outside {showFullImageDescription ? 'expanded' : ''}">
+      {mediaItem.description}
+    </div>
+    {/if}
     <div class="media-controls-outside" on:click="onMediaControlsClick(event)">
-      {#if canPinchZoom}
+      {#if hasDescription }
+        <IconButton
+          className="media-control-button"
+          svgClassName="media-control-button-svg"
+          pressable={hasDescription}
```

Comment:
This `pressable={hasDescription}` is unnecessary; we're already in an `if hasDescription` block.

```suggestion
          pressable={true}
```

Reply:
Oh! Yeah good catch - this is left over from an earlier iteration that didn't have the if blocks here.